Formula/vgmstream.rb

```diff
@@ -1,7 +1,7 @@
 class Vgmstream < Formula
   desc "Library for playing streamed audio formats from video games"
   homepage "https://hcs64.com/vgmstream.html"
-  url "http://svn.code.sf.net/p/vgmstream/code", :revision => 1040
+  url "https://gitlab.kode54.net/kode54/vgmstream.git", :tag => "r1040"
```

Comment:
Sorry, it was unverified. I've moved this update to #10898 (including the missing `:revision`). Will follow up there.